ice_setup/ice.py

```diff
@@ -567,6 +575,7 @@ def create_repo_file(cls, template_name, repo_url, gpg_url, file_name=None, **kw
             contents = template.format(
                 gpg_url=gpg_url,
                 repo_url=repo_url,
+                gpg_check=1 if use_gpg else 0,
```

Comment:
why not just use `kw` here instead of changing the signature, like:

    gpg_check=kw.get('use_gpg', 0)